src/constant.jl

```diff
@@ -5,54 +5,49 @@
 export Constant
 export vexity, evaluate, sign, conic_form!
 
-struct Constant <: AbstractExpr
+ispos(x::Real) = x >= 0
+ispos(v::AbstractVecOrMat{<:Real}) = all(ispos, v)
+isneg(x::Real) = x <= 0
+isneg(v::AbstractVecOrMat{<:Real}) = all(isneg, v)
+
+_size(x::Number) = (1, 1)
+_size(x::AbstractVector) = (length(x), 1)
+_size(x::AbstractMatrix) = size(x)
+
+_sign(x::Union{Complex,AbstractVecOrMat{<:Complex}}) = ComplexSign()
+_sign(x::Value) = ispos(x) ? Positive() :
```

Comment:
I think this would be clearer as:
```
function _sign(x::Value)
    if ispos(x)
        Positive()
    elseif isneg(x)
        Negative()
    else
        NoSign()
    end
end
```